phalcon/Helper/Str.zep

```diff
@@ -361,9 +361,9 @@ class Str
     final public static function includes(string! needle, string! haystack) -> bool
     {
         if function_exists("mb_strpos") {
-            return (bool) mb_strpos(haystack, needle);
+            return (bool) false !== mb_strpos(haystack, needle);
         } else {
-            return (bool) strpos(haystack, needle);
+            return (bool) false !== strpos(haystack, needle);
```

Comment:
Why cast to `bool` if proper condition always returns `bool`?

Reply:
I think that was remnant of the old code. Removing the `(bool)` will do what we need.

Reply:
Ok, I'll do that on Monday